src/main/scala/io/iohk/ethereum/utils/BlockchainConfig.scala

```diff
@@ -122,6 +124,8 @@ object BlockchainConfig {
     val allowedMinersPublicKeys = readPubKeySet(blockchainConfig, "allowed-miners")
 
     val ecip1099BlockNumber: BigInt = BigInt(blockchainConfig.getString("ecip1099-block-number"))
+    val capabilities: List[Capability] =
+      blockchainConfig.getStringList("capabilities").asScala.toList.map(Capability.from)
```

Comment:
I guess we should also check that the capability in the config is indeed a capability implemented by Mantis ?